src/core.cpp

```diff
@@ -63,9 +104,21 @@ void CVC::ChooseActions() {
             }
 
             //enumerate and score actions
+            std::vector<std::unique_ptr<Action>> actions = this->decision_engine_->EnumerateActions(*this, character.get());
+
             //choose one
             double choice = rand() / (RAND_MAX + 1.);
```

Comment:
it seems like using rand is discouraged, plus this code is kind of ugly. does that make sense to you?

Reply:
Yeah, I think the new rand stuff is better.  I'd put a std::mt19937_64 as a member variable on the CVC object, and then make a std::uniform_real_distribution.